### PR TITLE
Update GitHub Learning Event Info (Fixes #65)

### LINKED ISSUE #65: BIG BIG ISSUE
https://github.com/PurdueIEEE/IEEE-Website/blob/master/learning/index.php#L106
Time: Friday, Sept.30, Phys 223, 6-8pm
Mentors: Prof.Lu, Aditya, Brandon
Helpers: Yuqin, Ben, Sid, Justin, Michael

Also please at somewhere mention this github is something with prof.Lu and his ECE 264 class

### PR DESCRIPTION
**Pull request by stwrt**

### CODE REVIEW

File: learning/index.php [hunk 1]
```diff
@@ -103,12 +103,12 @@
...
+                    <li>Friday, September 30th in Phys 223 from 6:00 to 8:00 PM</li>
                 </ul>
                 <p><strong>Contact for additional questions and help:</strong></p>
                 <ul>
-                    <li>Matt Molo: mmolo@purdue.edu</li>
-                    <li>Tim Vincent: tv@purdue.edu</li>
+                    <li>Brandon Stewart: stewa174@purdue.edu</li>
+                    <li>Aditya Vaidyam: avaidyam@purdue.edu</li>
                 </ul>
                 <p>Matt will have office hours for additional help on the following Thursday, from 11:30 to 1:20 PM in EE 014.</p>
```
Comment (mattmolo) [inline learning/index.php]: No I won't.

File: learning/index.php [hunk 2]
```diff
@@ -103,12 +103,12 @@
                 <h2>Git + GitHub</h2>
```
Comment (mattmolo) [inline learning/index.php:103]: This should be moved above to upcoming events. Also Add Prof. Lu and a note that it is for ECE 264 students.

**Review by mattmolo — CHANGES_REQUESTED** [review]

File: learning/index.php (hunk 1 above)
Comment (stwrt) [reply learning/index.php]: ????

Comment (avaidyam) [reply learning/index.php]: @brstew Matt won't have office hours...

**Review by mattmolo — APPROVED** [review]

**Pull request merged**